addons/survey/views/survey_templates.xml

```diff
@@ -412,139 +410,140 @@
     <template id="question_simple_choice" name="Question: simple choice">
         <t t-set="answer_line" t-value="answer_lines.filtered(lambda line: line.suggested_answer_id)"/>
         <t t-set="comment_line" t-value="answer_lines.filtered(lambda line: line.value_char_box)"/>
-        <div class="row o_survey_form_choice"
+        <t t-set="answers_contains_image" t-value="question.suggested_answer_ids.filtered(lambda x : x.value_image)"/>
+        <t t-set="use_half_columns" t-value="survey.questions_layout == 'page_per_question' and answers_contains_image"/>
+        <div class="row g-2 o_survey_form_choice"
              t-att-data-name="question.id"
              data-question-type="simple_choice_radio">
             <t t-set="item_idx" t-value="0"/>
-            <div t-attf-class="col-lg-12 d-flex flex-wrap">
-                <t t-set="has_correct_answer" t-value="scoring_display_correction and any(label.is_correct for label in question.suggested_answer_ids)"/>
-                <t t-foreach='question.suggested_answer_ids' t-as='label'>
-                    <t t-set="item_idx" t-value="label_index"/>
-                    <t t-set="answer_selected" t-value="answer_line and answer_line.suggested_answer_id.id == label.id"/>
-                    <t t-set="is_correct" t-value="label.is_correct"/>
-
-                    <!--Used for print mode with corrections -->
-                    <t t-set="answer_class" t-if="not has_correct_answer" t-value="''" />
-                    <t t-set="answer_class" t-elif="is_correct" t-value="'bg-success'" />
-                    <t t-set="answer_class" t-elif="not is_correct" t-value="'bg-danger'" />
+            <t t-set="has_correct_answer" t-value="scoring_display_correction and any(label.is_correct for label in question.suggested_answer_ids)"/>
+            <t t-foreach='question.suggested_answer_ids' t-as='label'>
+                <t t-set="answers_contains_image" t-value="question.suggested_answer_ids.filtered(lambda x : x.value_image)"/>
+                <t t-set="item_idx" t-value="label_index"/>
+                <t t-set="answer_selected" t-value="answer_line and answer_line.suggested_answer_id.id == label.id"/>
 
+                <!--Used for print mode with corrections -->
+                <t t-set="answer_class" t-value="'' if not has_correct_answer else 'bg-success' if label.is_correct else 'bg-danger'"/>
+
+                <div t-attf-class="col-sm-12 #{'col-lg-6' if use_half_columns and not survey_form_readonly else '' 'col-lg-6' if answer.is_session_answer and answers_contains_image  else ''}">
```

Comment:
There's something off here...
And this is repeated 4 times throughout the file? Maybe this class can be defined once for all cases? (haven't checked).

Reply:
Do you want to use `t-set` variable to define the class and use it in the template where it is needed?
Am I right, please correct me if I am wrong.

Reply:
> Do you want to use t-set variable to define the class and use it in the template where it is needed?

Yes, if it can be defined in one place and then used in multiple places, it's better.

(You did not comment on that, so to be sure you noticed, the "off-looking" thing is the multiple 'col-lg-6' and probably undesired string `'' 'col-lg-6'` which is the same as 
```suggestion
                <div t-attf-class="col-sm-12 #{'col-lg-6' if use_half_columns and not survey_form_readonly else 'col-lg-6' if answer.is_session_answer and answers_contains_image  else ''}">
```

Which means `'col-lg-6' if x and not y else 'col-lg-6' if z and w else ''`